front/app/modules/commercial/insights/admin/components/TopBar/ProjectsDropdown.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { useState } from 'react';
+
+// styling
+import styled from 'styled-components';
+import { colors, fontSizes } from 'utils/styleUtils';
+
+// components
+import {
+  Dropdown,
+  DropdownListItem,
+  Icon,
+} from '@citizenlab/cl2-component-library';
+import Button from 'components/UI/Button';
+
+// intl
+import { injectIntl } from 'utils/cl-intl';
+import { InjectedIntlProps } from 'react-intl';
+import messages from './messages';
+
+// hooks
+import useProject from 'hooks/useProject';
+import useLocalize from 'hooks/useLocalize';
+
+// utils
+import { isNilOrError } from 'utils/helperUtils';
+
+const DropdownWrapper = styled.div`
+  color: ${colors.label};
+  font-size: ${fontSizes.small}px;
+
+  button {
+    font-weight: 500;
+    &:hover {
+      color: black;
+    }
+  }
+`;
+
+const StyledIcon = styled(Icon)`
+  width: 10px;
+  margin-left: 8px;
+`;
+
+interface ProjectTitleProps {
+  projectId: string;
+}
+
+const ProjectTitle = ({ projectId }: ProjectTitleProps) => {
+  const project = useProject({ projectId });
+  const localize = useLocalize();
+
+  if (isNilOrError(project)) return null;
+
+  const openProjectPage = () => {
+    window.open(`/projects/${project.attributes.slug}`, '_blank');
+  };
+
+  return (
+    <DropdownListItem onClick={openProjectPage}>
```

Comment:
I would really recommend making this a link vs handling it with `onClick`. It's better for accessibility. The user menu is an example of a dropdown with links if that's helpful.

Reply:
Good point, done :+1: